simulators/backend/genericbackend.py

```diff
@@ -108,6 +109,18 @@ def set_filename(self, filename):
     def convert_data():
         pass
 
+    def set_enable(self, feed1, feed2):
+        if feed1 not in range(self.max_sections / 2):
+            raise BackendException("feed1 out of range")
+        if feed2 not in range(self.max_sections / 2):
+            raise BackendException("feed2 out of range")
+        self.current_sections = [
+            feed1 * 2,
+            feed1 * 2 + 1,
```

Comment:
When there is a multiplication and addition, the [PEP8](https://peps.python.org/pep-0008/#other-recommendations) recommends to use the following style (no space between operands and operator with precedence):

`feed1*2 + 1  # no space between feed1 and 1`

If you have already written the style `feed1 * 2 + 1` in all the rest of the code, than it is better to keep it as it is for consistency, instead of changing everything because of PEP8.

Reply:
Apparently, prospector sees the change in both lines `feed1*2` and `feed1*2 + 1` as wrong from a PEP8 perspective, even though the PEP8 link you reported says it should be correct. I'll stick with the version with which prospector does not complain.